src/main/java/org/terasology/manualLabor/systems/ShearingSystem.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.manualLabor.systems;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.engine.core.Time;
+import org.terasology.engine.entitySystem.entity.EntityManager;
+import org.terasology.engine.entitySystem.entity.EntityRef;
+import org.terasology.engine.entitySystem.event.ReceiveEvent;
+import org.terasology.engine.entitySystem.systems.BaseComponentSystem;
+import org.terasology.engine.entitySystem.systems.RegisterSystem;
+import org.terasology.engine.logic.characters.events.AttackEvent;
+import org.terasology.engine.logic.delay.DelayManager;
+import org.terasology.engine.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.engine.logic.inventory.ItemComponent;
+import org.terasology.engine.registry.In;
+import org.terasology.gestalt.assets.ResourceUrn;
+import org.terasology.manualLabor.components.ShearableComponent;
+import org.terasology.wildAnimals.event.AnimalSpawnEvent;
+
+import java.util.ArrayList;
+
+@RegisterSystem
+public class ShearingSystem extends BaseComponentSystem {
+    public static final int HITS_TO_SHEAR = 5;
```

Comment:
I'm wondering whether we want to go for the simple version first and just have a single hit with shears shear the sheep. We could follow up on this then in a later step (e.g. in combination with adding more advanced tools that reduce the required hits).
The benefit in my eyes would be fewer things that can go wrong and need to be tested, documented, etc. while already providing a functional feature that allows us to focus on polishing that first before working on more complex variants of/to the feature.

Reply:
Can we discuss this a bit in the next meeting? If things go according to schedule, maybe we can keep this in and if I happen to take more time than anticipated, we can keep it for later.

Reply:
it's one additional member of the component and some counting logic here :sweat_smile:
I'm very confident, that you'll be able to re-introduce that in no time, once we have the instant-shearing polished, documented and there's still time.
Might even be we say it's more important to start early with the animal spawning, as - in my mind at least - that's still a bit of a question mark regarding required effort to do this properly :sweat_smile: especially as I'd like this to be done with proper event-based ECS architecture.

Reply:
Yeah, it won't take much time to reintroduce. I will remove it for now then.